organization_token.go

```diff
@@ -13,6 +13,14 @@ import (
 // Compile-time proof of interface implementation.
 var _ OrganizationTokens = (*organizationTokens)(nil)
 
+type TokenType = string
```

Comment:
curious as to why you chose to go with type alias (`type TokenType = string`) vs  `type TokenType string`. The former allows you to pass any string type to a function that accepts TokenType as a function param, whereas the latter only allows to pass only `TokenType` type.  The latter is better for type safety

Reply:
Huh whoops... Because it was typo.

I didn't expect that would compile, but TIL!  Thanks for sharp eyes @mrinalirao !

Reply:
Fixed